index.js

```diff
@@ -77,11 +79,14 @@ app.get('/docs.json', (function() {
  * Set up endpoints
  */
 endpoints.forEach(function(path) {
-    console.log('Setting up:', path);
+  if(!fs.existsSync('./endpoints/' + path + '/index.js')){
```

Comment:
How about wrapping the `require` with a `try` instead, and output the error?

In addition to catching `index.js` not existing, this would error gracefully if there is an error initializing the endpoint. (syntax errors, etc)
Would also allow the endpoint to have a `package.json` describing a different `main` than `index.js`, but maybe we won't ever need/want this.

I guess we could `throw` the `catch`ed if we're not in production? (easier in development)